src/components/file-handler/FileHandler.jsx

```diff
@@ -22,6 +22,11 @@ export const FILE_HANDLER_COLORS_SET = {
   WHITE: 'white',
 };
 
+export interface AriaStatusLabelType {
```

Comment:
Let's use `type` instead of interface here. Interface in flow has similar use cases to type but much wider and intention might be different - interface is the only way to use `implement` keyword on classes level or to merge types defined in different files. I would stick to using type if we don't need interface.

Also think it's internal type and we don't need to export it.

Reply:
ok :)